schema updated

joshuakarthik2005 opened the pull request:
I have attached the schema diagram file in the documentation page

venkatesh21bit commented on combined.py:
What is this and why is it in python?

venkatesh21bit commented on contents/docs/getting-started/schema-diagram/index.mdx:
Add this to the Route configuration file and add some text explaining the schema diagram inside the mdx

joshuakarthik2005 commented:
It isn't in python I added a mdx file under docs and added the pdf file
that contained the image in it.
It worked in my laptop
Will check on it again

On Wed, Jan 15, 2025, 12:16 K Venkatesh ***@***.***> wrote:

> Assigned #4 <https://github.com/venkatesh21bit/modernERP/pull/4> to
> @joshuakarthik2005 <https://github.com/joshuakarthik2005>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/venkatesh21bit/modernERP/pull/4#event-15934435471>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJBAHTOV6IRNP6JGP7KCUBT2KX73TAVCNFSM6AAAAABVGORTYSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJVHEZTINBTGU2DOMI>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

venkatesh21bit commented:
There is a python file called combined.py

venkatesh21bit commented:
U have to add the route in the route.config file to see the changes

joshuakarthik2005 commented:
I didn't add that I only created one file index.mdx under a folder i
created schema

On Wed, Jan 15, 2025, 12:29 K Venkatesh ***@***.***> wrote:

> There is a python file called combined.py
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/venkatesh21bit/modernERP/pull/4#issuecomment-2591786255>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJBAHTLDIS2MILEDORVE3RT2KYBNZAVCNFSM6AAAAABVGORTYSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOJRG44DMMRVGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

joshuakarthik2005 commented:
Will check on it again

On Wed, Jan 15, 2025, 12:31 JOSHUA KARTHIK ***@***.***>
wrote:

> I didn't add that I only created one file index.mdx under a folder i
> created schema
>
> On Wed, Jan 15, 2025, 12:29 K Venkatesh ***@***.***> wrote:
>
>> There is a python file called combined.py
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/venkatesh21bit/modernERP/pull/4#issuecomment-2591786255>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BJBAHTLDIS2MILEDORVE3RT2KYBNZAVCNFSM6AAAAABVGORTYSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKOJRG44DMMRVGU>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
